airbyte-cdk/python/airbyte_cdk/sources/declarative/datetime/datetime_parser.py

```diff
@@ -25,6 +27,8 @@ def parse(self, date: Union[str, int], format: str):
         # See https://stackoverflow.com/a/4974930
         if format == "%s":
             return datetime.datetime.fromtimestamp(int(date), tz=datetime.timezone.utc)
+        elif format == "%ms":
+            return self._UNIX_ZERO_DT + datetime.timedelta(milliseconds=int(date))
```

Comment:
we can't use datetime.fromtimestamp like we do for `%s`. Instead we can add a timedelta defined in milliseconds.